app/views/stages/_commands.html.erb

```diff
@@ -67,6 +68,13 @@
       return false;
     });
 
+    $('#commands pre.no_access').dblclick(function(){
+      var $warning = $('<div class="cannot-edit">Need admin access for the project, that owns this command.</div>')
+      $(this).after($warning);
+      $warning.fadeOut(3000);
+      return false;
+    });
+
```

Comment:
kinda copy-paste, can you create a function that adds them and then call it with selector + message ?